package com.xaquare.xquarebackoffice.global.error.exception

class XquareException(
val errorCode: ErrorCode
) : RuntimeException()
